src/components/userComponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,557 @@
+                </svg>
+              </Link>
+            </div>
+          </div>
+        </div>
+        <hr className="border-silver w-[322px] m-auto h-[.5px] mt-5 mb-8" />
+        <div className="bg-scary-forest flex justify-around text-center text-ivory font-[Lato] gap-4 active:text-[#bcc0bb]">
+          <div className="flex flex-col gap-4">
+            <Link href="/hoursAdmissionPage">
+              {' '}
+              <p> Hours & Locations </p>{' '}
+            </Link>
+            <Link href="/siteMapPage">
+              {' '}
+              <p> Site Maps </p>{' '}
+            </Link>
+            <Link href="/featuredToursPage">
+              {' '}
+              <p> Featured Tours </p>{' '}
+            </Link>
+          </div>
+          <div className="flex flex-col gap-4">
+            <Link href="/exhibitsPage">
+              {' '}
+              <p> Exhibits </p>{' '}
+            </Link>
+            <Link href="/newsFeedPage">
+              {' '}
+              <p> News </p>{' '}
+            </Link>
+            <Link href="/spotlightPage">
+              {' '}
+              <p> Wildlife Spotlights </p>{' '}
+            </Link>
+          </div>
+        </div>
+        <div className="w-full pt-6 gap-4 bg-scary-forest">
+          <Link href="https://phs-spca.org">
+            <div className="w-[200px] m-auto px-4 py-1.5 bg-hunterGreen rounded-lg">
+              <div className="text-center text-silver text-xs font-normal font-['Lato'] active:text-[#bcc0bb]">
+                Learn more about PHS/SPCA
+              </div>
+            </div>
+            <div className="mt-4 text-zinc-500 text-center text-xs font-normal font-['Lato']">
+              Copyright © 2024 Peninsula Humane Society & SPCA{' '}
+            </div>
+          </Link>
+          <Link
+            href="https://phs-spca.org/privacy-policy/"
+            className="mt-1 pb-10 text-center flex justify-center text-zinc-500 text-xs font-normal font-['Lato'] underline"
+          >
+            Privacy Policy
+          </Link>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+/**
+ *
+ * @param root0 web component
+ * @param root0.inputValueName name
+ * @param root0.inputValueEmail email
+ * @param root0.subscribed whether you subscribed
+ * @param root0.handleNameChange change name func
+ * @param root0.handleEmailChange change email func
+ * @param root0.handleSubmit what to do when submit email name
+ * @param root0.showError error
+ * @param root0.errorMsg error
+ * @returns web footer
+ */
+function WebInput({
```

Comment:
@jxmoose 
Missing subscribe prompt on web?
<img width="1305" alt="image" src="https://github.com/calblueprint/phs/assets/73627326/58d71333-a299-4f42-8189-0a968ee63955">
<img width="495" alt="image" src="https://github.com/calblueprint/phs/assets/73627326/b2c8a94c-66f3-47e6-a419-5be86a5cce6c">

Reply:
i think @cluurie  only wanted the subscribe on the home screen for web so can't do it in my component